1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,26 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+function quoteGenerator(){
```

Comment:
Nice layout of code here, with querySelectors -> appends -> eventListeners. Very readable